timesketch/lib/sigma_util.py

```diff
@@ -181,6 +182,18 @@ def get_sigma_rule(filepath, sigma_config=None):
 
     sigma_backend = sigma_es.ElasticsearchQuerystringBackend(sigma_conf_obj, {})
 
+    try:
+        sigma_rules_paths = get_sigma_rules_path()
+    except ValueError:
+        sigma_rules_paths = None
+
+    if not filepath.lower().endswith('.yml'):
+        return None
```

Comment:
the returns section says that you return a JSON representation... here you are returning none... that's not JSON

Reply:
ack, this now raises an Exception instead.